compiler/src/iree/compiler/Codegen/LLVMGPU/KernelConfig.cpp

```diff
@@ -830,6 +830,16 @@ setAttentionVectorDistributionConfig(IREE::GPU::TargetAttr target,
   attrs.emplace_back(StringAttr::get(context, "reduction"),
                      b.getI64ArrayAttr(reductionTileSizes));
 
+  SmallVector<NamedAttribute, 2> qkAttrs;
+  SmallVector<NamedAttribute, 2> pvAttrs;
+
+  qkAttrs.emplace_back(StringAttr::get(context, "attention_qk_matmul"),
+                       UnitAttr::get(context));
+  pvAttrs.emplace_back(StringAttr::get(context, "attention_pv_matmul"),
+                       UnitAttr::get(context));
+
+  auto qkAttrDict = DictionaryAttr::get(context, qkAttrs);
+  auto pvAttrDict = DictionaryAttr::get(context, pvAttrs);
```

Comment:
[optional, can be cleaned in a follow-up] so we have an OpBuilder declared in the function. These can be simplified by using [these methods](https://github.com/llvm/llvm-project/blob/main/mlir/include/mlir/IR/Builders.h#L110-L121). I marked it optional because I don't want to make additional changes in the file. It is a noise for review, IMO.

E.g.,

```suggestion
  SmallVector<NamedAttribute, 2> qkAttrs;
  SmallVector<NamedAttribute, 2> pvAttrs;

  qkAttrs.emplace_back(b.getNamedAttr("attention_qk_matmul", b.getUnitAttr());
  pvAttrs.emplace_back(b.getNamedAttr("attention_pv_matmul", b.getUnitAttr());

  auto qkAttrDict = b.getDictionaryAttr(qkAttrs);
  auto pvAttrDict = b.getDictionaryAttr(pvAttrs);
```

Reply:
I cleaned up the ones i changed, but i see what you mean, this kernel config logic can be cleaned up.